libs/accounts/src/lib/accounts-table.spec.tsx

```diff
@@ -24,47 +25,104 @@ const singleRow: AccountFieldsFragment = {
     symbol: 'tBTC',
     decimals: 5,
   },
+  available: '125600000',
+  used: '125600000',
+  deposited: '125600000',
 };
 const singleRowData = [singleRow];
 
 describe('AccountsTable', () => {
   it('should render successfully', async () => {
-    await act(async () => {
-      const { baseElement } = render(<AccountsTable data={[]} />);
-      expect(baseElement).toBeTruthy();
-    });
+    const { baseElement } = render(
+      <AccountTable rowData={[]} onClickAsset={() => null} />
+    );
+    expect(baseElement).toBeTruthy();
   });
 
   it('should render correct columns', async () => {
-    act(async () => {
-      render(<AccountsTable data={singleRowData} />);
-      await waitFor(async () => {
-        const headers = await screen.getAllByRole('columnheader');
-        expect(headers).toHaveLength(4);
-        expect(
-          headers.map((h) =>
-            h.querySelector('[ref="eText"]')?.textContent?.trim()
-          )
-        ).toEqual(['Asset', 'Type', 'Market', 'Balance']);
-      });
+    let headers: HTMLElement[];
+    await act(async () => {
+      render(
+        <AccountTable rowData={singleRowData} onClickAsset={() => null} />
+      );
+    });
+    await waitFor(async () => {
+      headers = screen.getAllByRole('columnheader');
+      expect(headers).toHaveLength(6);
+      expect(
+        headers?.map((h) =>
+          h.querySelector('[ref="eText"]')?.textContent?.trim()
+        )
+      ).toEqual(['Asset', 'Deposited', 'Used', '', '', '']);
     });
   });
 
   it('should apply correct formatting', async () => {
-    act(async () => {
-      render(<AccountsTable data={singleRowData} />);
-      await waitFor(async () => {
-        const cells = await screen.getAllByRole('gridcell');
-        const expectedValues = [
-          'tBTC',
-          singleRow.type,
-          'BTCUSD Monthly (30 Jun 2022)',
-          '1,256.00000',
-        ];
-        cells.forEach((cell, i) => {
-          expect(cell).toHaveTextContent(expectedValues[i]);
-        });
+    await act(async () => {
+      render(
+        <AccountTable rowData={singleRowData} onClickAsset={() => null} />
+      );
+    });
+    await waitFor(async () => {
```

Comment:
chore (blocking): `waitFor` shouldn't be required here.

Reply:
removed all waitFor